app/views/govuk_publishing_components/components/_taxonomy_navigation.html.erb

```diff
@@ -0,0 +1,84 @@
+<% if local_assigns[:items] && !items.blank? %>
+
+  <aside class='gem-c-taxonomy-navigation' data-module='track-click' role='complementary'>
+    <h2 class='gem-c-taxonomy-navigation__heading'><%= t("components.taxonomy_navigation.related_content") %></h2>
+    <% items.each_with_index do |item, item_index| %>
+      <div data-track-count="sidebarTaxonSection">
+        <h3 class='gem-c-taxonomy-navigation__heading'>
+          <%=
+            link_to_if(
+              item[:url],
+              item[:title],
+              item[:url],
+              class: 'gem-c-taxonomy-navigation__link',
+              data: {
+                track_category: 'relatedLinkClicked',
+                track_action: "#{item_index + 1}",
+                track_label: item[:url],
+                track_options: {
+                  dimension28: items.length.to_s,
+                  dimension29: item[:title],
+                },
+              },
+            )
+          %>
+        </h3>
+
+        <p class='gem-c-taxonomy-navigation__taxon-description'>
+          <%= item[:description] %>
+        </p>
+
+        <% item[:related_content] ||= [] %>
+        <% if item[:related_content].any? %>
+          <nav class='gem-c-taxonomy-navigation__navigation' role='navigation'>
+            <ul class='gem-c-taxonomy-navigation__list'>
+              <% item[:related_content].each_with_index do |related_item, related_content_index| %>
+                <li class='gem-c-taxonomy-navigation__item'>
+                  <%=
+                    link_to(
+                      related_item[:title],
+                      related_item[:link],
+                      data: {
+                          track_category: 'relatedLinkClicked',
+                          track_action: "#{item_index + 1}.#{related_content_index + 1}",
+                          track_label: related_item[:link],
+                          track_options: {
+                            dimension28: item[:related_content].length.to_s,
+                            dimension29: related_item[:title],
+                          }
+                      },
+                      class: 'related-link gem-c-taxonomy-navigation__link',
+                    )
+                  %>
+                </li>
+              <% end %>
+            </ul>
+          </nav>
+        <% end %>
+      </div>
+    <% end %>
+
+    <% local_assigns.except(:items).each do |section_name, section| %>
+      <% if section && !section.blank? %>
+        <div>
+          <h3 class='gem-c-taxonomy-navigation__section-heading'>
+            <%= t("components.taxonomy_navigation.#{section_name}") %>
+          </h3>
+          <nav class='gem-c-taxonomy-navigation__navigation' role='navigation'>
```

Comment:
We're note handling having a long list of values (this normally applies to world locations), like the related navigation component which puts them behind a show more toggle

Reply:
Added here https://github.com/alphagov/govuk_publishing_components/pull/253/commits/9e5d8fb0b08e72481dd8002ee796fa9af17227d7, I need to write a feature covering this.

Reply:
@steventux Worth adding an example to the documentation 👍 Also, the indentation in the view doesn't look quite right now?